### PR TITLE
Add gradle cache known issue

### LINKED ISSUE #1185: Known Issues: Gradle Cache
I've heard a few reports of corrupt Gradle caches this year:

https://www.chiefdelphi.com/t/gradle-issue-on-fresh-issue-and-default-template/389519/8
https://www.chiefdelphi.com/t/wpilib-build-error/390971/16

We should add a note to the known issues page about how to solve this.

### PR DESCRIPTION
**Pull request by Daltz333**
Close #1185

### CODE REVIEW

**Review by prateekma — COMMENTED** [review]
I'm not sure if this actually fixes the issue: https://www.chiefdelphi.com/t/wpilib-build-error/390971/20?u=prateek_m

**Comment by Daltz333** [comment]
It's only some errors, also, I commited on that post.

**Comment by Daltz333** [comment]
I've updated the known-issue document with just the gradlerio specific problem.

**Comment by Daltz333** [comment]
Thoughts Austin?

File: source/docs/yearly-overview/known-issues.rst [hunk 1]
```diff
@@ -8,6 +8,19 @@ This article details known issues (and workarounds) for FRC\ |reg| Control Syste
 Open Issues
 -----------
 
+Invalid build due to missing GradleRIO
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+**Issue:** Rarely, a users gradle cache will get broken and they will get shown errors similar to the following:
```
Comment (AustinShalit) [inline source/docs/yearly-overview/known-issues.rst:14]:
```suggestion
**Issue:** Rarely, a user's Gradle cache will get broken and they will get shown errors similar to the following:
```

File: source/docs/yearly-overview/known-issues.rst [hunk 2]
```diff
@@ -8,6 +8,19 @@ This article details known issues (and workarounds) for FRC\ |reg| Control Syste
...
+
+.. code-block:: console
+
+   Could not apply requested plugin [id: ‘edu.wpi.first.GradleRIO’, version: ‘2020.3.2’] as it does not provide a plugin with id ‘edu.wpi.first.GradleRIO’
+
+**Workaround:**
+
+Delete your gradle cache located under ``~$USER_HOME/.gradle``. Windows machines may need to enable the ability to `view hidden files <https://support.microsoft.com/en-us/windows/view-hidden-files-and-folders-in-windows-10-97fbc472-c603-9d90-91d0-1166d1d9f4b5>`__. This issue has only shown up on Windows so far, please `report <https://github.com/wpilibsuite/frc-docs/issues/new>`__ this issue if you get it on an alternative OS.
```
Comment (AustinShalit) [inline source/docs/yearly-overview/known-issues.rst:22]:
```suggestion
Delete your Gradle cache located under ``~$USER_HOME/.gradle``. Windows machines may need to enable the ability to `view hidden files <https://support.microsoft.com/en-us/windows/view-hidden-files-and-folders-in-windows-10-97fbc472-c603-9d90-91d0-1166d1d9f4b5>`__. This issue has only shown up on Windows so far. Please `report <https://github.com/wpilibsuite/frc-docs/issues/new>`__ this issue if you get it on an alternative OS.
```

**Review by AustinShalit — CHANGES_REQUESTED** [review]

**Review by AustinShalit — APPROVED** [review]

**Pull request merged**